package.json

```diff
@@ -18,7 +18,7 @@
     "clean": "lerna run --parallel clean",
     "commit": "cz",
     "publish:all": "lerna publish from-package",
-    "test": "jest --verbose --detectOpenHandles",
+    "test": "jest --verbose --detectOpenHandles --watchAll --no-cache",
```

Comment:
Added `--watchAll --no-cache` to fix this [issue](https://github.com/NibiruChain/ts-sdk/actions/runs/7179437222/job/19549611045).